airbyte-integrations/connectors/source-aircall/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 912eb6b7-a893-4a5b-b1c0-36ebbe2de8cd
-  dockerImageTag: 0.2.12
+  dockerImageTag: 0.3.0
```

Comment:
It would demand that you also add a change entry to `docs/integrations/aircall.md`

Reply:
Updated!